FlowCrypt/Core/DataManager.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  DataManager.swift
```

Comment:
Please put all files in FlowCrypt/Core as before. 

Core folder should only contain code related to our JavaScript core library that we refuse across apps. Nothing else. 

You can use FlowCrypt/Common for the rest.

Reply:
Din't know about it. Will fix